src/modules/ROOT/pages/changelog/202209/06.json

```diff
@@ -0,0 +1,206 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/752/commits/6856d91ea09d3a78750acb60c94a456e4a09d426"
+      ]
+    },
+    {
+      "what": [
+        "being unable to export an user that had a suspended or deleted role assigned to it."        
+      ],
+      "how": [
+        "the export logic was resolving unresolved resources on the fly and it was not excluded and this edge case caused the server to panic."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/14326713e"
+      ]
+    },
+    {
+      "what": [
+        "missing support for filtering, sorting, and paging over federation nodes."        
+      ],
+      "how": [
+        "by enabling filtering, sorting, and paging for federation nodes."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/22f49b031"
+      ]
+    },
+    {
+      "what": [
+        "hiding of calendar in calendar block when switching from page builder to view page and vice versa."
+      ],
+      "how": [
+        "by replacing throttle with async/await in the CalendarBase component."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/4124f8fee4db4e546f836ff2ab9b2222cbb5c666"
+      ]
+    },
+    {
+      "what": [
+        "searching a record list if it does not have any fields selected in the configurator."
+      ],
+      "how": [
+        "by making sure a record list with default fields uses those instead of the selected ones to search from."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/0f03f9c06381f5e63dd70d85c65c70c294babcdc"
+      ]
+    },
+    {
+      "what": [
+        "filtering for multi-value number field with contains or not contains criteria."
+      ],
+      "how": [
+        "by making sure to convert numbers into string format while building a query for the selected filter."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/7185611647d9960bcaf7153f5b5726f3f584fbfa"
+      ]
+    },
+    {
+      "what": [
+        "when record list records per page was set to 0, the pagination information was not displayed properly."
+      ],
+      "how": [
+        "by adding a check for this case where the pagination information is generated."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/045da2eaa472952409abfacc1b8142515a02f0d2"
+      ]
+    },
+    {
+      "what": [
+      "filtering for bool type for equal and not equal operator."
+    ],
+      "how": [
+      "by adding custom val handler for bool type to convert it properly while parsing the query according to supported drivers."
+    ],
+      "refs": [
+      "https://github.com/cortezaproject/corteza/commit/1febba3ebd53fa4ee38d3b7fc49a28859e605633"
+    ]
+  }
+  ],
+
+  "removed": [
+    {
+      "what": [
+        "the bootstrap class text-truncate from the <b-list-group-item>."
```

Comment:
I'd consider removing this one.
I/we generally collapse such entries (generic ui tweaks, css changes, ...) under a cookie cutter "UI/ux improvements" entry. Since there aren't many here I'd just remove it